src/controllers/user.js

```diff
@@ -59,11 +59,18 @@ export const getAll = async (req, res) => {
 }
 
 export const updateById = async (req, res) => {
-  const { cohort_id: cohortId } = req.body
-
-  if (!cohortId) {
-    return sendDataResponse(res, 400, { cohort_id: 'Cohort ID is required' })
+  if (
```

Comment:
How does this work with empty strings? Is it possible for the user to set their bio to an empty string? The validation here should match whatever we require on registeration - in otherwords, if am able to register with an empty bio I should be able to update my profile with an empty bio.